testdata/p4_16_errors/default-values-operations-err.p4

```diff
@@ -12,9 +12,6 @@ struct S {
     bool   b;
 }
 
-S f<D>(in D data) {
```

Comment:
why delete this?

Reply:
Call to this function with default values is enabled now. I left this file in p4_16_errors by accident. I removed it and added a new test with this function in p4_16_samples.